AdminWebsite/AdminWebsite/ClientApp/src/app/booking/search-email/search-email.component.ts

```diff
@@ -140,12 +142,11 @@ export class SearchEmailComponent implements OnInit, OnDestroy {
 
     validateEmail() {
         const pattern = Constants.EmailPattern;
-        this.isValidEmail =
-            this.email &&
-            this.email.length > 2 &&
-            this.email.length < 256 &&
-            pattern.test(this.email) &&
-            this.email.indexOf(this.invalidPattern) < 0;
+        this.isValidEmail = this.email && this.email.length > 2 && this.email.length < 256 && pattern.test(this.email);
+
+        if (!this.isJudge) {
+            this.isValidEmail = this.email.indexOf(this.invalidPattern) < 0;
+        }
```

Comment:
Invalid pattern was to prevent courtroom accounts from being added on add-participants.

Reply:
Good one 👍